includes/class-wcs-object-data-cache-manager.php

```diff
@@ -222,4 +310,15 @@ protected function trigger_update_cache_hook_from_change( $object, $key, $change
 				break;
 		}
 	}
+
```

Comment:
Missing docblock

Reply:
@shendy-a8c I've fixed this in [8a9106f](https://github.com/Automattic/woocommerce-subscriptions-core/pull/383/commits/8a9106f403614d6b2252628ea71f85354c1a6e78)

Reply:
Thank you, @mattallan !